README.md

```diff
@@ -1,7 +1,7 @@
 libswiftnav
 ===========
 
-[![Build status][1]][2]
+libswfitnav/master: [![Build status][1]][2]
```

Comment:
Typo!

You could also have have `master`, since `libswiftnav` is redundant. 

Feel free to merge after that.